bundle/edu.gemini.pot/src/main/java/edu/gemini/spModel/target/env/GuideProbeTargets.java

```diff
@@ -366,6 +366,11 @@ public boolean primaryIsBagsTarget() {
     }
 
     /**
+     * Determine if the primary guide star exists but is not a BAGS target.
+     */
+    public boolean primaryIsManual() { return primaryTarget.exists(manualTargets::contains); }
```

Comment:
I think the best thing to do is to leave this as is right now, i.e. have BAGS create a duplicate target, and discuss with Andy and Bryan what they think is best, unless anyone else has any ideas.

There doesn't seem to be a UI consistent, obvious way to handle it to me.